beets/ui/commands.py

```diff
@@ -1479,6 +1480,19 @@ def move_items(lib, dest, query, copy, album, pretend, confirm=False):
             show_path_changes([(obj.path, obj.destination(basedir=dest))
                                for obj in objs])
     else:
+        if export:
+
```

Comment:
Could you please clean up this extra blank space, and instead add a brief comment about what's going on here?